frontend/server/src/Controllers/QualityNomination.php

```diff
@@ -826,13 +834,13 @@ private static function hasDuplicates(array $contents): bool {
     }
 
     /**
-     * @return array{nominations: list<array{author: array{name: null|string, username: string}, contents?: array{before_ac?: bool, difficulty?: int, quality?: int, rationale?: string, reason?: string, statements?: array<string, string>, tags?: list<string>}, nomination: string, nominator: array{name: null|string, username: string}, problem: array{alias: string, title: string}, qualitynomination_id: int, status: string, time: \OmegaUp\Timestamp, votes: list<array{time: \OmegaUp\Timestamp|null, user: array{name: null|string, username: string}, vote: int}>}|null>, pager_items: list<array{class: string, label: string, page: int}>}
-     *
      * @omegaup-request-param int $offset
      * @omegaup-request-param int $rowcount
      * @omegaup-request-param mixed $status
      * @omegaup-request-param mixed $query
      * @omegaup-request-param mixed $column
+     *
+     * @return NominationList
```

Comment:
Aquí puedes agregar la anotación directamente:
```suggestion
     * @return array{nominations: list<NominationListItem|null>, pagerItems: list<PageItem>}
```

Eso mismo lo puedes agregar en L945

Reply:
listo